pages/api/attributes/[attributeId].ts

```diff
@@ -0,0 +1,69 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import {
+  getAttribute,
+  updateAttribute,
+  deleteAttribute,
+} from '../../../server/actions/Attributes'
+import { ApiError, Attribute } from '../../../utils/types'
+import { unstable_getServerSession } from 'next-auth'
+import { authOptions } from '../auth/[...nextauth]'
+
+// @route GET api/attributes/[attributeId] - Returns a single Attribute object given by a attributeId - Private
+// @route PUT api/attributes/[attributeId] - Updates an existing Attribute object (identified by categoryId) with a new Category object - Private
```

Comment:
change instances of 'category' to 'attribute'

Reply:
Fixed with f84b9c1!